pkg/scheduler/framework/plugins/podtopologyspread/common.go

```diff
@@ -87,14 +87,26 @@ func nodeLabelsMatchSpreadConstraints(nodeLabels map[string]string, constraints
 	return true
 }
 
-func filterTopologySpreadConstraints(constraints []v1.TopologySpreadConstraint, action v1.UnsatisfiableConstraintAction, enableMinDomainsInPodTopologySpread, enableNodeInclusionPolicyInPodTopologySpread bool) ([]topologySpreadConstraint, error) {
+func filterTopologySpreadConstraints(constraints []v1.TopologySpreadConstraint, podLabels map[string]string, action v1.UnsatisfiableConstraintAction, enableMinDomainsInPodTopologySpread, enableNodeInclusionPolicyInPodTopologySpread bool, enableMatchLabelKeysInPodTopologySpread bool) ([]topologySpreadConstraint, error) {
```

Comment:
can we just make the function a member of PodTopologySpread instead of passing all those flags? I worry we will mix them up since they are a list of back-to-back boolean parameters

Reply:
Done make it as the member function